### PR TITLE
Add code of conduct infrastructure

### PR DESCRIPTION
**Pull request by juliosantos**
This is still a work in progress.

I don't like the `redirect` trick, but it seemed the most obvious way to do this here. Open to suggestions.

### CODE REVIEW

File: app/components/App.jsx [hunk 1]
```diff
@@ -25,6 +26,7 @@ const App = () => (
     <Layout>
       <Switch>
         <Route exact path="/" component={anonymousRoute(Home)} />
+        <Route exact path="/conduct" component={anonymousRoute(Conduct, false)} />
```
Comment (hugopeixoto) [inline app/components/App.jsx]: If you don't want the redirect behaviour, just remove the `anonymousRoute` HOC. It's the only thing it does.

`component={Conduct}` should work, or am I missing something?

Comment (juliosantos) [reply app/components/App.jsx]: I think you're right and I might have violently derped here. Will check later.

File: app/components/pages/Conduct.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  render() {
+    const { t } = this.props;
+
+    return (
+      <div className="posts">
+        <section className="post">
+          <h1 className="post-title">{t("title")}</h1>
+
+          <ul>
+            {t("rules", {returnObjects: true}).map((rule, index) => (
```
Comment (jgradim) [inline app/components/pages/Conduct.jsx:17]: 👌

**Review by jgradim — APPROVED** [review]
:shipit:

**Pull request merged**